src/SlashCommands.tsx

```diff
@@ -660,7 +660,7 @@ export const Commands = [
             if (args) {
                 const cli = MatrixClientPeg.get();
 
-                const matches = args.match(/^(\S+)$/);
+                const matches = args.match(/^(@[^:]+:\S+)$/);
```

Comment:
Since the pattern is tightly bound, we could do without the capture group and just use `args` whenever a match is found. I thought I'd start with a more conservative change.